.github/workflows/build.yml

```diff
@@ -4,15 +4,17 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.17, 1.18, 1.18@latest, 1.19beta1, 1.19@dev-latest, latest, dev-latest]
-        os: [ubuntu-latest, macos-latest]
+        go-version: [1.17]
```

Comment:
Why 1.17?

Reply:
Just testing. So reduced matrix as I expected to fail anyways